ui-tests/src/test/resources/warnings_ng_plugin/source-view/SampleClassWithoutPackage.java

```diff
@@ -0,0 +1,3 @@
+public class Main {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The name of the outer type and the file do not match.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)